src/Translation_unit.ml

```diff
@@ -132,7 +135,12 @@ let parse_print (XUnit xunit) (conf : Conf.t) ~input_name ~input_file ic
     xunit.fmt source cmts_t conf ast fs ;
     Format.pp_print_newline fs () ;
     Out_channel.close oc ;
-    let fmted = In_channel.with_file tmp ~f:In_channel.input_all in
+    ( match Conf.line_endings conf input_file with
+    | `Windows when Sys.unix ->
+        let command = Format.sprintf "sed -i 's/$/\r/' %s" tmp in
+        ignore (Unix.system command)
```

Comment:
nit: In general I'd prefer to have type annotations on ignore for safety: `Unix.system command |> (ignore : Unix.process_status -> _)`

It would probably be good to give an understandable error message if this fails. In particular, users might not expect ocamlformat to run sed.